### PR TITLE
Re-design: About story page.

### PR DESCRIPTION
**Pull request by YuriGizra**
#213
Remove footer from about story page.

### CODE REVIEW

**Comment by YuriGizra** [comment]
@YaronMiro, ready.

File: app/_layouts/default.html [hunk 1]
```diff
@@ -40,24 +40,26 @@
...
-              5 Brenner St., Tel-Aviv, Israel
-            </div>
-            <div class="telephone">
-              <a href="tel:+97233731222">+972-3-3731222</a>
-            </div>
-            <div>
-              <a href="mailto:info@gizra.com" class="email">info@gizra.com</a>
-            </div>
-          </address>
+    {% if page.url != "/content/the-about-story" %}
```
Comment (YaronMiro) [inline app/_layouts/default.html]: Please change it 
`{% if page.url != "/content/the-about-story" %}` => `{% if page.exclude_footer == nil %}`
And add `exclude_footer: true` on the `the-about-story` yml configuration

**Comment by YuriGizra** [comment]
@YaronMiro, fixed

**Comment by YaronMiro** [comment]
Thanks merged manually.

**Pull request closed without merging**